internal/main

```diff
@@ -510,8 +510,24 @@ function post_build_preview {
 
 # Publishes the docs for Jenkins or for a dev.
 function publish {
-    [ -n "$JENKINS_URL" ] && activate_jenkins_env
-    echo "where does publish live? in docker? or just here, like it is in fe lambda?"
+    if [ -n "$JENKINS_URL" ]; then
+	activate_jenkins_env
+	MESSAGE="Test, build, and publish with Jenkins (build # $BUILD_NUMBER)"
+    else
+	MESSAGE="Manually published by $USER"
+    fi
+    git worktree add dragons gh-pages
+    rm -rf dragons/*
+    rsync -a docs/_build/html/ dragons/
```

Comment:
Should this be publishing the docs created by the test invoking the `html` function or docs built using `htmlvers`?

Reply:
The Jenkinsfile publish stage needs to be
```
stage("Release stuff") {
  sh './rax-docs htmlvers'
  sh './rax-docs publish'
}
```
That mimics the Jenkinsfile in the lambda docs.